Modules/include/VersionInfoProvider.h

```diff
@@ -0,0 +1,61 @@
+// SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
+// SPDX-License-Identifier: LGPL-3.0-or-later
+#pragma once
+
+#include "ApplicationModule.h"
+#include "ConfigReader.h"
+#include "ScalarAccessor.h"
+#include "VersionInfo.h"
```

Comment:
Please add a comment that this header file is auto-generated by the project template in the target project, hence all references to this (`namespace ChimeraTK::VersionInfo`) must be header-only and not included in the ApplicationCore .so file.